near/src/runtime.rs

```diff
@@ -96,8 +107,23 @@ impl NightshadeRuntime {
                     .collect(),
             )
             .expect("Failed to start Epoch Manager"),
+        ));
+        let shard_tracker = ShardTracker::new(
```

Comment:
so I would make ShardTracker part of near cargo, move account_id_to_shard_id there (for now until we hash out the dynamic shard number) and call it from here.